server/plugin/api.go

```diff
@@ -285,6 +287,38 @@ func (p *Plugin) handleGetSubscriptions(w http.ResponseWriter, r *http.Request)
 	p.writeJSON(w, subscriptionList)
 }
 
+func (p *Plugin) handleSubscriptionNotifications(w http.ResponseWriter, r *http.Request) {
+	body, err := serializers.SubscriptionNotificationFromJSON(r.Body)
+	if err != nil {
+		p.API.LogError("Error in decoding the body for creating notifications", "Error", err.Error())
+		p.handleError(w, r, &serializers.Error{Code: http.StatusBadRequest, Message: err.Error()})
+		return
+	}
+
+	channelID := r.URL.Query().Get("channelID")
+	if channelID == "" {
+		p.API.LogError(constants.ChannelIDRequired)
+		p.handleError(w, r, &serializers.Error{Code: http.StatusBadRequest, Message: constants.ChannelIDRequired})
+		return
+	}
+
```

Comment:
```suggestion

if !model.IsValidId(channelID) {
```
Add one more condition for checking if channelID is valid or not